utils/ValidateReadmeTable.py

```diff
@@ -0,0 +1,62 @@
+import os
+import re
+import sys
+from typing import Final
+
+CHECK_ID_PATTERN: Final[str] = r"(PWR\d+|PWD\d+|RMK\d+)"
+CHECKS_DIR: Final[str] = "Checks/"
+README_FILE: Final[str] = "README.md"
+TABLE_MARKER: Final[str] = "|"
+
+
+def extractTableFromReadme(readmeFile: str) -> list[str]:
+    with open(readmeFile, "r") as file:
+        lines = file.readlines()
+
+    tableLines = [
+        line for line in lines if line.strip() and line.startswith(TABLE_MARKER)
+    ]
+
+    return tableLines
```

Comment:
Not sure what's the purpose of `line.strip()` but it shouldn't change anything with the `startswith` after.

Also, seems like it can be more... succinct:

```suggestion
    with open(readmeFile, "r") as file:
        return  [
            line for line in file.readlines() if line.startswith(TABLE_MARKER)
        ]
```

Reply:
> Not sure what's the purpose of line.strip() but it shouldn't change anything with the startswith after.

Indeed, sorry for the oversight. The `strip()` condition was from the initial implementation of the script, but it doesn't make sense anymore. It's been removed!

> Also, seems like it can be more... succinct:

Done as well.